### DISCUSSION TITLE
Unsquished Max Hyperview with Max Lens Mod 2.0 in 8:7?

### DISCUSSION THREAD

**Discussion by tallen11** [gopro/labs#760] [Q&A] [answered]
Hello,

Is there/will there be support for recording in 8:7 while using the Max Lens Mod 2.0 w/ GoPro Hero 12? It seems like it should be possible since Max Hyperview is an 8:7 image squished into 16:9. I'd love to get the biggest, widest, tallest FOV possible (like you get with max hyperview) in an undistorted 8:7 aspect ratio.

Thanks,
Tate

**Comment by dnewman-gpsw** [comment, marked as answer]
It would seem possible, based on your presented logic.  A 3840x3360 8:7 MLM2 mode.  Not something Labs can add, as it can't add video modes.

Put a request in here https://gopro.com/en/us/info/static/feedback

**Reply by tallen11** [reply]
Done, thanks for the reply.
